toolkits/global/packages/global-ratings-survey/README.md

```diff
@@ -0,0 +1,134 @@
+# Global Ratings Survey
+
+```text
+Component README follows Elements’ template, covering:
+- [Description]
+- [Demo]
+- When to use and when not to use this component
+- How it works
+- Installation
+- Template
+- Research on this component
+
+## Description
+- Designed to be used at multiple points in a journey
+- Collects information in a passive way
+- Designed to be comparable for data and collect any instances of major errors on our websites continuously
+- Designed to be unobtrusive - can be placed any page at any point in a user journey
+## When to use this component
+- Use this component for the CSAT method
+- Used in combination with other survey tools
+  - Could be used in combination with an additional survey at the end of a task / exit survey
+## When not to use this component
+- Do not use this component for NPS. NPS  is a different use-case - not an either or. If you need to use NPS - it should be placed at the end / exit of a task.
+## How to use (setup)
+Who to contact
+The question can be not changed
+- Surveymonkey
+- Discuss with Ben on the backend
+- Research ops?
+- Document what we discuss with Roland on how to get setup
+- Where it goes on the page (e.g. right hand column may only be used in contexts where the content is very long)
+```
+
+## Usage
+This documents how to implement this component on your website. If your project does not use Elements yet you can still use this component, see refer to the `Projects that do not use Elements` section below.
+### Install
+Install the Global Ratings Survey package:
+```shell
+npm install @springernature/global-ratings-survey
+```
+You will also need to ensure you have its dependencies installed. These are Brand Context and Global Forms:
+```shell
+npm install @springernature/brand-context
+```
+```shell
+npm install @springernature/global-forms
```

Comment:
All these instructions will install the latest versions of all these components which is NOT what you want to be doing. You need to tailor these to specify the exact version of each of these e.g. `npm install @springernature/global-ratings-survey@0.0.1`

This is probably the place to add the caveat about this being an anti-pattern that I mentioned in a previous comment

Reply:
Good point, thanks!